mysql-test/t/a3.test

```diff
@@ -0,0 +1,79 @@
+--source include/have_innodb.inc
```

Comment:
All main.a* have been removed. Instead, a new MTR test case "innodb.xtradb_compressed_columns_ibd_sizes" has been added. This test shows that it is possible to achieve a better compression ratio using compression dictionaries.